packages/ignite-element/src/examples/xstate/styles.css

```
@@ -2,3 +2,4 @@
 
 @source "./index.html";
 @source "./*.ts";
+@source "./*.tsx";
```

Stylelint (17.4.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)